app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training-course.md

```diff
@@ -109,29 +96,29 @@ Your teacher training course might be provided by:
 * a school or group of schools (sometimes referred to as school-led training, or an apprenticeship) 
 * [Teach First](https://www.teachfirst.org.uk/) (a charitable organisation)  
 
-Once you’ve found a course that you’re interested in, you can talk to the training provider before you apply. For example, you might want to find out: 
+Once you’ve found a course you’re interested in you can talk to the training provider before you apply. For example, you might want to find out: 
```

Comment:
```suggestion
Once you’ve found a course you’re interested in, you can talk to the training provider before you apply. For example, you might want to find out: 
```

Reply:
I find this hard to understand without a comma splitting up - is it best practice not to have one?